lib/utils.php

```diff
@@ -22,20 +32,21 @@ function seekPOSTorGET( $key, $default = NULL, $type = NULL )
     {
         if( isset( $type ) )
             settype( $_POST[ $key ], $type );
-        return $_POST[ $key ];
+        $retval = $_POST[ $key ];
     }
     else if( $_GET !== FALSE && array_key_exists( $key, $_GET ) )
     {
         if( isset( $type ) )
             settype( $_GET[ $key ], $type );
-        return $_GET[ $key ];
+        $retval = $_GET[ $key ];
     }
     else
     {
         if( isset( $type ) )
             settype( $default, $type );
-        return $default;
+        $retval = $default;
```

Comment:
if `$type` is not string, `ascii_only` should not be called.

Reply:
even with strings `ascii_only` should not be called. It affected the forums and rich presence, and they non-ascii chars.

I reverted this PR. Better to find a cleaner solution than this badly coded makeshift.